src/Commands/Lists/GetLargeListOperationStatus.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Management.Automation;
+
+using Microsoft.SharePoint.Client;
+
+using PnP.PowerShell.Commands.Base.PipeBinds;
+using PnP.PowerShell.Commands.Model.SharePoint;
+
+namespace PnP.PowerShell.Commands.Lists
+{
+    [Cmdlet(VerbsCommon.Get, "PnPLargeListOperationStatus")]
+    public class GetLargeListOperationStatus : PnPWebCmdlet
+    {
+        [Parameter(Mandatory = true)]
+        public Guid ListId;
```

Comment:
Can you please modify the code to use `ListPipeBind` and rename this parameter as `Identity`, for reference you can check `Set-PnPList` cmdlet ? 

So, something like this:

```
public ListPipeBind Identity;
```

and then in the code:

```
var list = Identity.GetList(CurrentWeb);
var operation = ClientContext.Web.GetListOperation(list.Id, OperationId);
```

Reply:
By the time when the customer runs the command, the list could've been deleted and I just tested 
`var list = Identity.GetList(CurrentWeb)` won't find the list.